src/lib.rs

```diff
@@ -11,6 +11,8 @@ compile_error!(
      and 'tls' is one of 'native-tls' and 'rustls'."
 );
 
+pub use sqlx_core as core;
```

Comment:
```suggestion
```
The API of `sqlx_core` isn't meant to be used directly.

Reply:
But it is required to implement own Executor?

Reply:
Well it is needed for Describe... Not sure why you need to hide it

Reply:
We don't care to support custom implementations of `Executor` right now. A proper implementation would require significant access to internal APIs.

`describe()` is mainly used for the macros and the API is not considered stable as we may add or remove from it to suit our purposes there. That's why it's hidden.

Reply:
In my case it just wrapper to provide contextual transaction
But if thats your position then i understand, i shouldnt rely on this lib